aries-site/src/data/wcag/notification.json

```diff
@@ -0,0 +1,512 @@
+  },
+  {
+    "rule": "1.2.7",
+    "label": "Extended Audio Description (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.8",
+    "label": "Media Alternative (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.9",
+    "label": "Audio-only (Live)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.1",
+    "label": "Info and Relationships",
+    "status": "failed",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.2",
+    "label": "Meaningful Sequence",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.3",
+    "label": "Sensory Characteristics",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.4",
+    "label": "Orientation",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.5",
+    "label": "Identify Input Purpose",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.6",
+    "label": "Identify Purpose",
+    "status": "failed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.1",
+    "label": "Use of Color",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.2",
+    "label": "Audio Control",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.3",
+    "label": "Contrast (Minimum)",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.4",
+    "label": "Resize Text",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.5",
+    "label": "Images of Text",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.6",
+    "label": "Contrast (Enhanced)",
+    "status": "failed",
+    "notes": "AAA criteria"
+  },
+  {
+    "rule": "1.4.7",
+    "label": "Low or No Background Audio",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.8",
+    "label": "Visual Presentation",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.9",
+    "label": "Images of Text (No Exception)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.10",
+    "label": "Reflow",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.11",
+    "label": "Non-text Contrast",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.12",
+    "label": "Text Spacing",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.13",
+    "label": "Content on Hover or Focus",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.1.1",
+    "label": "Keyboard",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.1.2",
+    "label": "No Keyboard Trap",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.1.3",
+    "label": "Keyboard (No Exception)",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.2.1",
+    "label": "Timing Adjustable",
+    "status": "failed",
+    "notes": ""
```

Comment:
Wait should this be conditional? It's available via prop: https://v2.grommet.io/notification#time

Reply:
For AA compliance: Avoid auto-dismiss toasts unless users can adjust or disable the timer.
Alternatively, switch toast behavior to:
→ "Remains until dismissed" OR
→ "Includes pause/dismiss options when focused"


users cant really adjust that? or would it be the case that someone implementing can?
Disappears automatically after a few seconds
- Has no visible controls to extend/pause
- Offers no user setting to change timing

Reply:
Hmm -- we can see what @jcfilben thinks next week. My feeling is that the grommet component offers the ability to. So it would be "conditional" for the product team to provide the user some option in their "settings".